ast/src/call.rs

```diff
@@ -33,8 +33,11 @@ impl SourceSegmentHolder for Call<'_> {
 #[segment_holder]
 #[derive(Debug, Clone, PartialEq, DebugPls)]
 pub struct ProgrammaticCall<'a> {
```

Comment:
With the introduction of the `MethodCall` structure, you should refactor this to `ProgrammaticFunctionCall`